featuretools/utils/recommend_primitives.py

```diff
@@ -0,0 +1,244 @@
+]
+
+
+# TODO: Support multi-table
+def get_recommended_primitives(
+    entityset: EntitySet,
+    include_time_series_primitives: bool,
+    excluded_primitives: List[str] = DEFAULT_EXCLUDED_PRIMITIVES,
+) -> List[str]:
+    """Get a list of recommended primitives given an entity set.
+
+    Description:
+        This function works by first getting a list of valid primitives withholding any primitives specified in `excluded_primitives` that could be applied to a single-table EntitySet.
+        Secondly, engineered features are created for non-numeric fields and are checked for non-uniqueness. If the feature is non-unique, it is added to the recommendation list.
+        Then, numeric fields are checked for skewness. Depending on how skew a column is `square_root` or `natural_logarithm` will be recommended.
+        Lastly if `include_time_series_primitives` is specified as `True`, `Lag` will always be recommended,
+        as well as all Rolling and Expanding primitives if numeric columns are present.
+
+    Args:
+        entityset (EntitySet): EntitySet that only contains one dataframe.
+        include_time_series_primitives (bool): Whether or not time-series primitives should be considered.
+        excluded_primitives (List[str]): List of transform primitives to exclude from recommendations.
+
+    Note:
+        The main objective of this function is to recommend primitives that could potentially provide important features to the modeling process.
+        Non-numeric primitives do a great job in mainly serving as a way to extract information from origin features that may essentially be meaningless by themselves (e.g., NaturalLaguange, Datetime, LatLong).
+        That is why they are the main focus of this function. Numeric transform primitives are very case-by-case dependent and therefore it is hard to mathmatically quantify which should be recommended.
+        Therefore, we decided to only look at transform primitives that address skewed numeric columns as this is a standard and quantifiable transformation step. The only exception to this rule being
+        for time series problems. Because there are so few primitives that are only applicable for time series, we recommend users try all of them.
+
+    Note:
+        This function currently only works for single table and will only recommend transform primitives.
+    """
+    es_dataframe_list = list(entityset.dataframe_dict)
+
+    if len(es_dataframe_list) == 0:
+        raise IndexError("No DataFrame in EntitySet found. Please add a DataFrame.")
+    if len(es_dataframe_list) > 1:
+        raise IndexError(
+            "Multi-table EntitySets are currently not supported. Please only use a single table EntitySet.",
+        )
+
+    target_dataframe_name = es_dataframe_list[0]
+
+    recommended_primitives = set()
+
+    if not include_time_series_primitives:
+        excluded_primitives += TIME_SERIES_PRIMITIVES
+    else:
+        recommended_primitives.add("lag")
```

Comment:
I think this could be removed and Lag would still get added, if valid, along with the other valid time series primitives

Reply:
I think this was left over from previous logic, good catch